guides/v2.1/install-gde/install/cli/install-cli-subcommands-admin.md

```diff
@@ -43,24 +48,72 @@ Where the following table defines parameters and values:
 |`--admin-password`|Magento administrator user password. The password must be at least 7 characters in length and must include at least one alphabetic and at least one numeric character. <br><br>We recommend a longer, more complex password. Enclose the entire password string in single quotes. For example, `--admin-password='A0b9%t3g'`.|Yes|
 |`--magento-init-params`|Add to any command to customize Magento initialization parameters<br/><br/>For example: `MAGE_MODE=developer&MAGE_DIRS[base][path]=/var/www/example.com&MAGE_DIRS[cache][path]=/var/tmp/cache`|No|
 
+Example of usage:
+
+```bash
+bin/magento admin:user:create --admin-firstname=John --admin-lastname=Doe --admin-email=j.doe@example.com --admin-user=j.doe --admin-password=A0b9%t3g
+```
+
+```terminal
+Created Magento administrator user named j.doe
+```
+
+If you do not specify any of required params Magento will ask about them in the CLI:
+
+```bash
+bin/magento admin:user:create
+```
+
+```terminal
+Admin user: John
+Admin password: 
+Admin email: j.doe.young@example.com
+Admin first name: John
+Admin last name: Doe Young
+```
```

Comment:
Hi @atwixfirster,
When we run command `bin/magento admin:user:create` then we get this line no 68 output for Magento 2.3.x version and 2.2.x. so can you please revert this changes from doc file v2.1 and create a file for v2.2 and v2.3 doc with above change.

Reply:
**done**

Thank you for notice, @ronak2ram ! 👍 

Could you please review now?

Reply:
Hi @atwixfirster 
thanks for the update.